docs/tutorial/part-one/index.md

```diff
@@ -1,390 +1,282 @@
 
-1.  Gatsby lets you add "inline styles" via a JavaScript style "prop" (we'll
-    learn about other styling options later).
+### 💡 Wait… HTML in our JavaScript?
 
-    Try replacing your page component with this:
+*If you’re familiar with React and JSX, feel free to skip this section.* If you haven’t worked with the React framework before, you may be wondering what HTML is doing in a JavaScript function. Or why we’re importing `react` but not using it anywhere. This hybrid “HTML-in-JS” is actually a syntax extension of JavaScript, for React, called JSX. You can follow along this tutorial without prior experience with React, but if you’re curious, here’s a brief primer…
 
-```jsx
-import React from "react";
+This is the JSX from the starter `/src/pages/index.js` file:
 
-export default () => <div style={{ color: `blue` }}>Hello Gatsby!</div>;
-```
 
-Change the color to "pink". Then to "tomato".
+    import React from 'react'
+    import Link from 'gatsby-link'
+    const IndexPage = () => (
+      <div>
+        <h1>Hi people</h1>
+        <p>Welcome to your new Gatsby site.</p>
+        <p>Now go build something great.</p>
+        <Link to="/page-2/">Go to page 2</Link>
+      </div>
+    )
+    export default IndexPage
 
-2.  Add some paragraph text.
+In pure JavaScript, it looks more like this:
 
-```jsx{5-6}
-import React from "react";
 
-export default () =>
- <div style={{ color: `tomato` }}>
-   <h1>Hello Gatsby!</h1>
-   <p>What a world.</p>
- </div>
-```
+    import React from 'react';
+    import Link from 'gatsby-link';
+    
+    const IndexPage = () => React.createElement(
+      'div',
+      null,
+      React.createElement(
+        'h1',
+        null,
+        'Hi people'
+      ),
+      React.createElement(
+        'p',
+        null,
+        'Welcome to your new Gatsby site.'
+      ),
+      React.createElement(
+        'p',
+        null,
+        'Now go build something great.'
+      ),
+      React.createElement(
+        Link,
+        { to: '/page-2/' },
+        'Go to page 2'
+      )
+    );
+    
+    export default IndexPage;
 
-3.  Add an image (in this case, a random one from Unsplash)
+(Now you can spot the use of the `'react'` import on several lines in the snippet above!) But wait. We’re writing JSX, right? How does the browser read that? The short answer: It doesn’t. Gatsby sites comes with tooling already set up to convert your source code into backward-compatible code that different browsers can reliably interpret. 
 
-```jsx{7}
-import React from "react";
+## Meet components: The starter header
 
-export default () =>
- <div style={{ color: `tomato` }}>
-   <h1>Hello Gatsby!</h1>
-   <p>What a world.</p>
-   <img src="https://source.unsplash.com/random/400x200" alt="" />
- </div>
-```
+If `/src/pages/index.js` generated our homepage, then where was our purple “Gatsby” header?
 
-Now your screen should look something like this:
+1. Open up `/src/components/Header/index.js`.
```

Comment:
Ok the starter is different now, which changes a lot of things about this tutorial :/

- `src/components/header.js` has replaced `src/components/Header/index.js`
- The title text is in `gatsby-config.js`
- `{children()}` is in `src/layouts/index.js` line 30

Reply:
1) Is there anything specific you want me to add about hot reloading? 

2) re: the "likely to be skipped" -- yeah, you're probably right. But I feel about it the same way I feel about https://github.com/gatsbyjs/gatsby/pull/5526#discussion_r191627224 <- that comment. It may be a lightbulb moment for people who are really trying to understand why it's working, rather than just accept the "magic"

3) Shouldn't take too long to make adjustments based on the changes to the starter, no worries